src/popup/components/dashboard/Card.vue

```diff
@@ -71,6 +87,7 @@ export default {
   background-size: cover;
   text-align: center;
   transition: background-color 0.2s;
+  position: relative;
```

Comment:
I suggest to use this patterns to organise CSS properties: https://9elements.com/css-rule-order/